src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -145,6 +145,12 @@ public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFil
                     details.Add(imdbId);
                 }
 
+                if(movie.YouTubeTrailerId.IsNotNullOrWhiteSpace())
+                {
+                    String[] tube = { "plugin://plugin.video.youtube/?action=play_video&amp;videoid=", movie.YouTubeTrailerId};
+                    details.Add(new XElement("trailer", string.Join("", tube)));
```

Comment:
Why make this a list and join? Aren’t we always just combining the two strings? Should be able use string interpolation